### DISCUSSION TITLE
Android SDK licence

### DISCUSSION THREAD

**Discussion by Chralu** [talsec/Free-RASP-Community#21] [Q&A] [answered]
Hello,

I read here and there that the FreeRasp SDK is distributed under MIT licence.
Indeed, [Flutter SDK is](https://github.com/talsec/Free-RASP-Flutter/blob/master/LICENSE), but it seems to rely on native SDK implementations.

And I cannot find Android SDK repository. 

Could anyone confirm under which licence are Android and iOS SDKs distributed ?

Thanks for your help 🙏

**Comment by syakymchuk** [comment, marked as answer]
Hi @Chralu 

Thank you for the question.  The Product is distributed under the Freemium business model and licensing. i.e. it is not fully opensource as explained [here](https://github.com/talsec/Free-RASP-Community#license). We have the freemium plan for freeRASP SDK and the premium paid version for RASP+ SDK (with a number of extra security features and hardening for SDK resilience). 

Have a great day!

**Reply by Chralu** [reply]
Thank you for your fast and documented reply 🙏
